lib/hca/settings.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module HCA
+  module Settings
+    CONFIG = Rails.application.config_for(:health_care_application).freeze
```

Comment:
Assuming that yaml file never changes, might want to move this to an initializer so that the yaml file is loaded once at startup instead of many times during runtime.